databases/week1/dbWK1.sql

```diff
@@ -0,0 +1,70 @@
+--1.Find out how many tasks are in the task table
+SELECT count(*)
+FROM task; 
```

Comment:
You can also use count(0) instead of count(*) 
Its give a same result.

Reply:
thank you so much for your feedback, i will use it from next time...!